### PR TITLE
remove vulnerabilities from this repo

### PR DESCRIPTION
**Pull request by vdeturckheim**
As of https://github.com/nodejs/security-wg/issues/494 vulnerabilities are not handled by this repo anymore but in https://github.com/nodejs/security-advisories.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "test": "jest",
-    "report": "node tools/reporter",
-    "validate": "node tools/vuln_valid",
-    "create-npm-index": "node tools/create_index/create_npm_index.js",
-    "create-core-index": "node tools/create_index/create_core_index.js"
+    "report": "node tools/reporter"
```
Comment (lirantal) [inline package.json]: What did we decide eventually? we're moving out all the Pull Requests to be opened in the security-advisories repo too =, right? meaning we need to move this tool out to the other repo as well.

Comment (lirantal) [reply package.json]: FYI that it needs to move there with axios, inquirer, jest and yargs probably.

**Comment by sam-github** [comment]
what is the state of this? I look forward to being able to PR text changes to the processes without having to wait for a travis build!

**Comment by vdeturckheim** [comment]
@sam-github I will handle this before our next meeting on Monday and Travis would be history.

**Comment by fraxken** [comment]
Is this PR still active @vdeturckheim ? I see that the repo `security-advisories` is archived.

**Comment by vdeturckheim** [comment]
@fraxken we can close I think

**Pull request closed without merging**